README.md

```diff
@@ -37,7 +35,27 @@ npm install
 node server.js
 ```
 
-
 # Contributing
 
-We are very open to new contributors. If you want to contribute to this project, and don't know where to start, look at the [open issues](https://github.com/jrtechs/github-graphs/issues). Once you know what you want to work on, comment on the issue and file a pull request. 
\ No newline at end of file
+We are very open to new contributors. If you want to contribute to this project, and don't know where to start, look at the [open issues](https://github.com/jrtechs/github-graphs/issues). Once you know what you want to work on, comment on the issue and file a pull request.
+
+# API Reference
+
+`https://github-graphs.com/api/friends/<username>`
+
+- `https://api.github.com/users/<username>`
+- `https://api.github.com/users/<username>/followers`
+- `https://api.github.com/users/<username>/following`
+- `https://api.github.com/users/<username>/orgs`
+
+`https://github-graphs.com/api/repositories/<username>`
+
+- `https://api.github.com/users/<username>/repos`
+
+`https://github-graphs.com/api/org/users/<username>`
+
+- `https://api.github.com/orgs/<username>/members`
+
+`https://github-graphs.com/api/org/repositories/<username>`
```

Comment:
We should provide a high-level explanation for each API endpoint, like "returns all repositories that are owned by a GH organization"